keylime/tenant.py

```diff
@@ -692,16 +692,17 @@ def do_cvbulkinfo(self):
 
         response_json = Tenant._jsonify_response(response, print_response=False)
 
+        if response.status_code == 500:
+            raise UserError(f"Unexpected response from Verifier {self.verifier_ip} on port {self.verifier_port}")
+
         if response.status_code == 503:
-            logger.error(
-                "Cannot connect to Verifier at %s with Port %s. Connection refused.",
-                self.verifier_ip,
-                self.verifier_port,
+            raise UserError(
+                f"Cannot connect to Verifier at {self.verifier_ip} on port {self.verifier_port}. Connection refused."
```

Comment:
An existing issue: this should 'service is unavailable' rather than 'connection refused', which I find misleading.

https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/503

Reply:
Good point, I did not change this message, as it was not part of the original PR, but it is a small fix.